packages/nimble-components/src/table/tests/table.stories.ts

```diff
@@ -36,14 +36,28 @@ const simpleData = [
     }
 ] as const;
 
-const simpleDataIdFieldName = 'firstName';
+const firstNames = ['John', 'Sally', 'Joe', 'Michael', 'Sam'];
+const lastNames = ['Davidson', 'Johnson', 'Abraham', 'Wilson'];
+const colors = ['Red', 'Blue', 'Green', 'Yellow'];
+const largeData = [];
+for (let i = 0; i < 10000; i++) {
+    largeData.push({
+        id: i.toString(),
+        firstName: firstNames[i % firstNames.length],
+        lastName: lastNames[i % lastNames.length],
+        favoriteColor: colors[i % colors.length],
+        quote: null
```

Comment:
nit: could be nice to have some data that is unique across all rows.
```suggestion
        quote: `I'm number ${i}!`
```

Reply:
Updated